index.html

```diff
@@ -1,92 +1,211 @@
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+    <div class="card">
+      <h1>Profesional Art Printing Data</h1>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. 
+        has been the industry's standard</p>
+        <div class="card-btn">
+          <button>html</button>
+          <button>bootstrap</button>
+          <button>Ruby</button>
+        </div>
+        <button class="cardbtn2">See Project</button>
+    </div>
+  </section>
+  <section class="about">
+    <div class="card-aboutme">
+      <h1>About me</h1>
+      <p>Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button id="card2btn">Get My Resume</button>
+    </div>
+    <div id="line2"></div>
+    <div class="skills">
+    
+      <h2><span id="diamond"></span> Language</h2>      
+      <h3>Javascript</h3>
+      <h3>Ruby</h3>
+      <h3>Html</h3>
+      <h3>Css</h3>
+      <h2><span id="rectangle"></span>Frameworks</h2>
+      <h3>Bootstrap</h3>
+      <h3>Ruby on Rails</h3>
+      <h3>RSPec</h3>
+      <h3>CapyBara</h3>
+      <h3>Selenium</h3>
+      <h2><span id="circle"></span> Skills</h2>
+      <h3>Codekit</h3>
+      <h3>Github</h3>
+      <h3>Codepen</h3>
+      <h3>Gitlab</h3>
+      <h3>Terminal</h3>

```

Comment:
- [ ] The overall design looks very good, let's use `ul` and `li` tags for these three lists, this will make your code even better e.g:

```
<ul>
    <li>Javascript</li>
    <li>Ruby</li>
    <li>HTML</li>
<ul>
```